components/solutions/UseCases.vue

```diff
@@ -1,18 +1,23 @@
 <template>
     <div class="container mb-5">
-        <Section :title="title" :subtitle="subtitle">
+        <Section
```

Comment:
Header "Use Cases" is in uppercase (due to p.overline styling) while it's not on Figma